### PR TITLE
Secure URLs: http --> https

### PR DESCRIPTION
**Pull request by tehnick**
+ Update few outdated links.

### CODE REVIEW

**Comment by tehnick** [comment]
Ping?

File: COPYING [hunk 1]
```diff
@@ -1,7 +1,7 @@
                     GNU GENERAL PUBLIC LICENSE
                        Version 3, 29 June 2007
 
- Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
+ Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
```
Comment (janvidar) [inline COPYING]: This change is invasive.

**Comment by janvidar** [comment]
This change is too big and invasive. Especially since it touches on almost all files.

File: COPYING (hunk 1 above)
Comment (tehnick) [reply COPYING]: Fixed. Any additional comments?

**Comment by tehnick** [comment]
I have reverted change for `COPYING` file, but what is the problem with all other files?
Now is 2019 year: almost all developers of websites are worried about users security.
More other, modern web browsers are going to mark all http connections as insecure since this year or already do this.

**Comment by tehnick** [comment]
For example: https://security.googleblog.com/2018/02/a-secure-web-is-here-to-stay.html

**Comment by janvidar** [comment]
I don't mind the fact that the web has moved on to HTTPS for the most part, but I don't like mixing commits.
So, one commit that touches all copyright headers, and another commit that touches everything else is preferred.

**Comment by tehnick** [comment]
Sorry, but I failed to see the mixing of commits. Could you show exact places?

I have prepared few independent PRs specially for not mixing the changes.

**Comment by tehnick** [comment]
BTW, how about other my PRs?

**Comment by janvidar** [comment]
About mixing commits:
I don't like one commit that touches virtually *all* source code files that contain the copyright header to be mixed with various other changes to miscellaneous files that just happen to refer to an http URL.
These should be two logical commits.

File: COPYING (hunk 1 above)
Comment (Kcchouette) [reply COPYING]: The GNUGPL3 has https: https://www.gnu.org/licenses/gpl-3.0.txt

**Comment by tehnick** [comment]
@janvidar Ok, I'll split this into two commits. But personally I still do not see the difference: `git show` transparently shows all changes + all changes are linked by one topic.

**Comment by tehnick** [comment]
Sorry for delay: I forgot about this PR.

@janvidar 
I did necessary git magic and have split all changes into two commits. Please review this PR again.

**Comment by tehnick** [comment]
Ping?

**Comment by tehnick** [comment]
Any additional comments?

**Comment by janvidar** [comment]
Thanks

**Pull request merged**